fili-core/src/main/java/com/yahoo/bard/webservice/web/DataApiRequest.java

```diff
@@ -1098,52 +1193,56 @@ protected DateTime getCurrentDate(DateTime dateTime, TimeGrain timeGrain) {
     }
 
     /**
+     * Extract valid sort direction.
+     *
+     * @param columnWithDirection  Column and its sorting direction
```

Comment:
Since this is a String, we should definitely add more detail to this comment that indicates the expected format(s).

/CanBeInAFollowOnPr